contribute/process/commit_template

```diff
@@ -0,0 +1,16 @@
+# <50 character summary of the change
+
+# 1. Why is this change necessary ?
+
```

Comment:
Give a sample answer that is detailed.
The committer is expected to reword the sentence or delete & create his own.